File: back-end/skeleton/dotnet/src/DotnetSkeleton.API/DotnetSkeleton.IdentityModule.Domain/Entities/BaseEntities/BaseEntity.cs
namespace DotnetSkeleton.IdentityModule.Domain.Entities.BaseEntities
{
    public class BaseEntity
    {
        public int? CreatedBy { get; set; }
        public int? UpdatedBy { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime? UpdatedDate { get; set; }
        public bool IsDeleted { get; set; } = false;
    }
}

File: back-end/skeleton/nodejs-express/src/entities/base.entity.js
const BaseColumnSchemaPart = {
    // _id: {                   // Primary column is used for MongoDB
    //     objectId: true,
    //     name: '_id'
    // },
    createdDate: {
        name: 'CreatedDate',
        type: 'datetime',
        createDate: true
    },
    updatedDate: {
        name: 'UpdatedDate',
        type: 'datetime',
        updateDate: true
    },
    createdBy: {
        name: 'CreatedBy',
        type: 'int'
    },
    updatedBy: {
        name: 'UpdatedBy',
        type: 'int'
    },
    isDeleted: {
        name: 'IsDeleted',
        type: 'tinyint',
        default: 0
    }
};

export default BaseColumnSchemaPart;